include/zephyr/bluetooth/hci.h

```diff
@@ -1488,6 +1499,56 @@ struct bt_hci_cp_le_ext_create_conn {
 	struct bt_hci_ext_conn_phy p[0];
 } __packed;
 
+#define BT_HCI_OP_LE_SET_PER_ADV_SUBEVENT_DATA  BT_OP(BT_OGF_LE, 0x0082)
+struct bt_hci_cp_le_set_pawr_subevent_data_element {
+	uint8_t subevent;
+	uint8_t response_slot_start;
+	uint8_t response_slot_count;
+	uint8_t subevent_data_length;
+	uint8_t subevent_data[0];
+} __packed;
+
+struct bt_hci_cp_le_set_pawr_subevent_data {
+	uint8_t adv_handle;
+	uint8_t num_subevents;
+	struct bt_hci_cp_le_set_pawr_subevent_data_element subevents[0];
```

Comment:
As I read it, the spec says the data is column-major. Pseudo code for illustration:
```suggestion
	uint8_t subevent[num_subevents];
	uint8_t response_slot_start[num_subevents];
	uint8_t response_slot_count[num_subevents];
	uint8_t subevent_data_length[num_subevents];
	uint8_t subevent_data[num_subevents];
```

Reply:
The spec writes the layout a bit weirdly, but it is row-major. You can take a look at the extended create connection command with the initiating_phys for something similar.

Reply:
Discussed OOB. EBQ tests say it's in row-major. We'll assume that's correct.